### PR TITLE
UI checklist: mobile

### PR DESCRIPTION
**Pull request by martinklepsch**
Various mobile UI cleanups

To review see the individual things to check here:
https://trello.com/c/gwJ1Q2u9/372-collected-ui-issues

Safari mobile issues:
https://trello.com/c/B5ygHR7Q

Also read the comments on that card why certain things haven't been implemented

- [x] **missing**: animation for previous/next inside an expanded topic
- [x]  improve visual balance between icons and title

### CODE REVIEW

File: src/open_company_web/lib/utils.cljs [hunk 1]
```diff
@@ -522,14 +524,12 @@
...
-        integer-string (clojure.string/replace int-part #"\B(?=(\d{3})+(?!\d))" ",")]
-    (if-not (nil? dec-part)
-      (str integer-string "." dec-part)
-      integer-string)))
+(def +usd-fmt+
+  (-> (NumberFormat. nf/Format.CURRENCY "USD" nf/CurrencyStyle.LOCAL)
+      (.setMinimumFractionDigits 0)))
+
+(defn ^:export thousands-separator [number]
+  (.format +usd-fmt+ number))
```
Comment (martinklepsch) [inline src/open_company_web/lib/utils.cljs]: remove export metadata

**Comment by bago2k4** [comment]
@belucid this is ready now, i added here the animation for prev and next revision for both mobile and big web (my fault, i didn't see the big web branch)

**Comment by belucid** [comment]
Reviewing.

**Comment by belucid** [comment]
- [ ] During Prev/Next animation (Desktop Web) the X close icon jumps right then left again

**Comment by bago2k4** [comment]
@belucid there were a couple of obvious issues but i could not replicate the one about the jumpy X that you mentioned here. can you retry and see if my last commits fixed that too? thanks

**Comment by belucid** [comment]
@bago2k4 the jumpy x is gone BTW

**Pull request merged**